components/FeatureHighlight/src/MDCFeatureHighlightViewController.m

```diff
@@ -57,6 +59,38 @@ - (nonnull instancetype)initWithHighlightedView:(nonnull UIView *)highlightedVie
   return self;
 }
 
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  [self setupFeatureHighlightView];
+}
+
+- (void)setupFeatureHighlightView {
```

Comment:
Do you expect someone to call this method other than viewDidLoad? If not, please keep the logic in viewDidLoad.